docs/manual/java/guide/services/code/docs/services/test/ServiceTestModule.java

```diff
@@ -75,6 +78,7 @@ public void bindServices(ServiceBinding<?>... serviceBindings) {
     // Bind the resolved services
     binder.bind(ResolvedServices.class).toProvider(new ResolvedServicesProvider(allServiceBindings));
 
+    binder.bind(new TypeLiteral<List<Router>>(){}).toInstance(Collections.emptyList());
```

Comment:
That was a surprise. Our docs have its own version of `ServiceGuiceSupport`? Why?

There is an explanation in a comment, but I didn't get it. :-(